### DISCUSSION TITLE
Farming from big size clouds

### DISCUSSION THREAD

**Discussion by Arabrus** [Chia-Network/chia-blockchain#5547] [Support]
Pls tell me, can I plotting with WebDAV- clouds like google disk or dropbox or .... something alse. And maybe I can cut plot of couple parts and send to my cloud and after glue there? If it's possible

**Comment by BrandtH22** [comment]
@Arabrus , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - due to latency and cloud storage provider terms it is generally not recommended to farm from cloud storage.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
